src/components/SubscribeSection/SubscribeSection.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Platform } from "../../hooks/usePlatform";
+import {
+  Content,
+  EnvelopeIcon,
+  Form,
+  Heading,
+  Image,
+  StyledSubscribeSection,
+  Text,
+} from "./styles";
+import { Input } from "../Input";
+import { Button } from "../Button";
+import { Pencilicon2 } from "../../icons/Pencilicon2";
+
+interface Props {
+  platform: string;
+}
+
+export const SubscribeCard = ({ platform }: Props): JSX.Element => {
+  return (
+    <StyledSubscribeSection platform={platform}>

```

Comment:
Please try to use either "Container" or "Wrapper" for the first tag, when debugging UI bugs, usually first step is to check what is inside component's UI container or wrapper.
```suggestion
    <Container platform={platform}>

```